### DISCUSSION TITLE
VM Scaling set works with source image reference but not custom image

### DISCUSSION THREAD

**Discussion by harrypotter033** [aztfmod/terraform-azurerm-caf#1334] [General]
Hi All,
I have custom image in shared images/gallery I can see I have access. Also the gallery is in different resource but i connected using peering both resource group.  But i received error when i use custom_image_id

`Error: Error updating Linux Virtual Machine Scale Set "DEV2-blue-web" (Resource Group "rg-DEV2-blue"): compute.VirtualMachineScaleSetsClient#Update: Failure sending request: StatusCode=0 -- Original Error: Code="GalleryImageNotFound" Message="\"The gallery image /subscriptions/13193103131319371/resourceGroups/rg-DEV2-images/providers/Microsoft.Compute/galleries/sig_DEV2_common_msn/images/si_DEV2_common_msn_web_e is not available in westus2 region. Please contact image owner to replicate to this region, or change your requested region.\"" Target="imageReference"
│
│   with module.caf.module.virtual_machine_scale_sets["web-vmss"].azurerm_linux_virtual_machine_scale_set.vmss["linux"],
│   on .terraform/modules/caf/modules/compute/virtual_machine_scale_set/vmss_linux.tf line 61, in resource "azurerm_linux_virtual_machine_scale_set" "vmss":
│   61: resource "azurerm_linux_virtual_machine_scale_set" "vmss" {`

I have specified image id like below.
`            custom_image_id = "/subscriptions/3193103131319371/resourceGroups/rg-DEV2-images/providers/Microsoft.Compute/galleries/sig_DEV2_common_msn/images/si_DEV2_common_msn_web_e"`

Any idea if i am doing anything wrong.

**Comment by IamSureshRa** [comment]
hi, your vmss and custom image are in different locations, you can make the image available in multiple regions. https://docs.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries?tabs=azure-cli#can-i-replicate-my-image-versions-across-subscriptions
